catan-server-module/catan-server.js

```diff
@@ -745,18 +745,18 @@ exports.handleNameChangeAttempts = function (io, socket, previousName, name, uui
 exports.handleMonopoly = function (io, socket, currentRoom, monopolyData) {
   var resourceName = order[monopolyData.resource]
   var uuid = monopolyData.uuid
-  console.log('monopoly', resourceName, uuid)
-  var totalCards = 0
-  roomData[currentRoom].forEach(function (player) {
-    if (playerData[uuid].getPlayerIndex() !== player.getPlayerIndex()) {
-      totalCards += player.getCards().monopolyResource(monopolyData.resource)
-    }
-  })
   if (playerData[uuid].getCards().getResourceAmount(8) > 0) { // Can't use if have 0
```

Comment:
One option is to use a "failfast approach", i.e. 

```
  if (playerData[uuid].getCards().getResourceAmount(8) === 0) {
 // Can't use if have 0
return;
}
...
```

Reply:
I prefer when a method hits the end of the method body to failing fast